scripts/ecs-anywhere-install.sh

```diff
@@ -592,6 +600,75 @@ wait-agent-start() {
     fail
 }
 
+exec-setup() {
+    find-and-move-certs
+    install-ssm-binaries-for-exec
+}
+
+find-and-move-certs() {
+    CERTS_PATH="/var/lib/ecs/deps/execute-command/certs"
+    CERTS_FILES_LOCATIONS=("/etc/ssl/certs/ca-certificates.crt"
+      "/etc/pki/tls/certs/ca-bundle.crt"
+      "/etc/ssl/ca-bundle.pem"
+      "/etc/pki/tls/cacert.pem"
+      "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+      "/etc/ssl/cert.pem"
+    )
+
+    echo "Copying certs for exec"
+    certs=""
+    if [ -n "$CERTS_FILE" ] && [ -f "$CERTS_FILE" ]; then
+        certs=$CERTS_FILE
+    else
+        if [ -n "$CERTS_FILE" ]; then
+            echo "Provided certs file does not exist, looking for certs in known possible locations"
+        fi
+        for f in "${CERTS_FILES_LOCATIONS[@]}"
+        do
+            if  [ -f "$f" ]; then
+                certs=$f
+                break
+            fi
+        done
+    fi
+
+    if [ -z "$certs" ]; then
+        echo "Could not find certificates. Please generate TLS certificates, or provide a valid file path to them"
```

Comment:
Can this message ask the user to rerun the script with the --certs-file flag to provide the vaild certificate path?